src/components/ui4/ButtonUi4.tsx

```diff
@@ -221,7 +221,8 @@ const getStyles = cacheStyles((theme: Theme) => {
       // Reduce the bounds of a tertiary button so it doesn't appear to be too
       // far from other buttons
       alignSelf: 'center',
-      padding: 0,
+      paddingHorizontal: 0,
+      paddingVertical: 0,
```

Comment:
I thought we already chatted about this? No change in padding for tertiary

Reply:
Yes, this is only horizontal padding, and it was already like this in the GUI